tests/test_core_to_contrib.py

```diff
@@ -1236,15 +1236,15 @@
         'airflow.operators.papermill_operator.PapermillOperator',
     ),
     (
-        'airflow.operators.check_operator.CheckOperator',
+        'airflow.operators.sql.SQLCheckOperator',
```

Comment:
You need to add new entry in this file. Now you have corrected the entry that changes airflow.operators.presto_check_operator.PrestoCheckOperator => airflow.operators.sql.SQLCheckOperator, but we have no entry for airflow.operators.check_operator.CheckOperator => airflow.operators.sql.SQLCheckOperator. I have intentionally asked to add entries because it is the easiest way to check that you have done everything correctly.

Reply:
Sorry for misunderstand your instruction. I added new entries in the new commit.